src/app/shared/email-validator.directive.ts

```diff
@@ -0,0 +1,24 @@
+import {Directive, Input} from '@angular/core';
```

Comment:
It is good to move this validator into a new folder, e.g. `directives` or `validators`